rpc/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "libra-rpc"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+description = "Libra rpc endpoint"
+repository = "https://github.com/libra/libra"
+homepage = "https://libra.org"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+serde_json = "1.0.40"
+serde = { version = "1.0.99", default-features = false }
+warp = "0.2.1"
+futures = { version = "0.3.0", features = ["io-compat", "compat"] }
```

Comment:
You probably don't want/need the compat features turned on

Reply:
@bmwill : thanks for pointing that out. Needed it on one of iterations. Will clean it up!